lib/ansible/roles/mysql/vars/main.yml

```diff
@@ -5,5 +5,4 @@ mysql_packages:
   - libapache2-mod-auth-mysql
   - mysql-server-5.6
   - mysql-server-core-5.6
-  - php5-mysql
```

Comment:
Moved to `php` role, where we choose the appropriate package for the current version of php